src/instructlab/llamacpp/llamacpp_convert_to_gguf.py

```diff
@@ -48,6 +48,9 @@
     # Standard
     from typing import TypeAlias
 
+# ruff: noqa: SIM115
```

Comment:
I was told elsewhere (by @bjhargrave) that this file (and other files that originate from other repos) should not be "fixed" by linters and e.g. mypy checker. (I disagree btw, but that's beyond the point.) If that's so, this change should probably also *not* happen. (mlx_explore and lora_mlx were also mentioned, I believe.)

Reply:
I am just conscious about excess modifications to 3rd party source we copied in. Sure, add features and fix things that break us. But modifying the code to pass our views on code linting seems a bit over the top and makes it harder to merge in a change from upstream if we want.

Reply:
Makes sense. I'm going to exclude the files from Ruff for now.

We should consider to make vendored files into a `instructlab.vendor` package, so it is obvious that the files were vendored from 3rd party projects.